src/App.tsx

```diff
@@ -1,27 +1,61 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [allgoods, setAllGoods] = useState<Good[]>([]);
```

Comment:
you need to use camelCase here 
```suggestion
  const [allGoods, setAllGoods] = useState<Good[]>([]);
```